components/Footer/styles.css

```diff
@@ -33,11 +33,137 @@
   font-size: 1em;
 }
 
+.PreFooter {
```

Comment:
we're using SUIT's https://suitcss.github.io/ naming conventions for our css. Can you update your class names to follow the same?

so everything within `PreFooter` would be named like `PreFooter-getInTouch`